tensorflow_addons/rnn/cell.py

```diff
@@ -366,3 +366,471 @@ def _create_norm_layer(self, name):
             gamma_initializer=self.norm_gamma_initializer,
             epsilon=self.norm_epsilon,
             name=name)
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+# class LayernormSimpleRNNCell(SimpleRNNCell, LayerNormalization):
+class LayernormSimpleRNNCell(keras.layers.SimpleRNNCell):
```

Comment:
Let's be consistent with the cell above, use "LayerNorm" with N.

Reply:
https://github.com/tensorflow/addons/pull/841/commits/348567c15bbc7381317c761f69ef2d7151e205ba

Reply:
https://github.com/tensorflow/addons/pull/841/commits/022eef45a73829741eb2056505248b94f59a6590